Makefile.PL

```diff
@@ -80,6 +82,7 @@ my %requires = (
   'Mojo::JWT'                           => '0.07',
   'MojoX::Log::Log4perl'                => '0.04',
   'Mojolicious'                         => '8.42',
```

Comment:
This should be updated to reflect the major version bump.

Also need to update `MIN_PERL_VERSION` to at least match Mojo's (5.16.0 as of 8.50), although it might make sense to make it match what we do in docker.